client/my-sites/stats/stats-strings.js

```diff
@@ -86,14 +86,14 @@ export default function() {
 		} ),
 	};
 
-	statsStrings.podcastdownloads = {
-		title: translate( 'Podcasts', { context: 'Stats: title of module' } ),
-		item: translate( 'Episodes', { context: 'Stats: module row header for podcast.' } ),
+	statsStrings.filedownloads = {
+		title: translate( 'File Downloads', { context: 'Stats: title of module' } ),
+		item: translate( 'Files', { context: 'Stats: module row header for file downloads.' } ),
 		value: translate( 'Downloads', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Downloads', { context: 'Stats: module row header for number of downloads per podcast episode.'} )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?